tutorials/unifiers/src/test/java/com/example/myapexapp/ApplicationTest.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * Put your copyright and license info here.
+ */
+package com.example.myapexapp;
+
+import java.io.IOException;
+
+import javax.validation.ConstraintViolationException;
+
+import org.junit.Assert;
+
+import org.apache.hadoop.conf.Configuration;
+import org.junit.Test;
+
+import com.datatorrent.api.LocalMode;
+import com.example.myapexapp.Application;
+
+/**
+ * Test the DAG declaration in local mode.
+ */
+public class ApplicationTest {
+
+  // helper routine
+  private void go(final boolean useUnifier) throws Exception {
+    try {
+      LocalMode lma = LocalMode.newInstance();
+      Configuration conf = new Configuration(false);
+      conf.addResource(this.getClass().getResourceAsStream("/META-INF/properties.xml"));
+      if (useUnifier) {
+        conf.setBoolean("dt.application.MyFirstApplication.operator.range.prop.useUnifier",
+                        true);
+      }
+      lma.prepareDAG(new Application(), conf);
+      LocalMode.Controller lc = lma.getController();
+      lc.run(10000); // runs for 10 seconds and quits
```

Comment:
You could alternatively collect the output in the ToConsole operator (in a static field)?